<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion des m√©dicaments</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(195, 177, 161);
color: rgb(202, 124, 92);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>301</width>
      <height>361</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(0, 127, 255);
background-color: rgb(238, 215, 197);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="title">
     <string>Formulaire </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantite</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_quantite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>261</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>261</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
gridline color: rgb(0, 127, 255);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DISPO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>OUI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NON</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>711</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>liste medicament</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rechercher">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>119</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom medicament</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>520</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_supprimer">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>510</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugestion_des_medicamants">
    <property name="title">
     <string>gestion des medicamants </string>
    </property>
   </widget>
   <addaction name="menugestion_des_medicamants"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
